include/uv/errno.h

```diff
@@ -302,6 +302,12 @@
 # define UV__ENOTSUP (-4049)
 #endif
 
+#if defined(EOVERFLOW) && !defined(_WIN32)
+# define UV__OVERFLOW UV__ERR(EOVERFLOW)
+#else
+# define UV__OVERFLOW (-4026)
+#endif
+
```

Comment:
I think the entries in this file are ordered in descending assigned value (i.e. `-4026`) comes last. That would make it easier to see what the value the next added entry should take.

Reply:
These are also in alphabetical order. Should we update all the error codes underneath?

Reply:
`UV__OVERFLOW` should be `UV__EOVERFLOW`.

Reply:
> These are also in alphabetical order. Should we update all the error codes underneath?

? They're not in alphabetical order.

Reply:
Just noticed it, moved it to the bottom. :)